frontend/src/App.vue

```diff
@@ -134,19 +179,19 @@ export default {
   padding: 0 !important;
 }
 .bottom-buttons-col {
-  border-top: 2px solid  var(--vscode-editorWidget-background, #252526);
+  border-top: 2px solid var(--vscode-editorWidget-background, #252526);
   padding-right: 25px;
 }
 .bottom-buttons-col > .v-btn:not(:last-child) {
-    margin-right: 10px !important;
+  margin-right: 10px !important;
```

Comment:
try to avoid making formatting changes in unmodified code.
It makes reviewing harder...